packages/eslint-plugin/docs/rules/restrict-plus-operands.md

```diff
@@ -18,7 +18,6 @@ This rule reports when a `+` operation combines two values of different types, o
 ### ❌ Incorrect
 
 ```ts
-let foo = '5.5' + 5;
```

Comment:
Ah, but then the first line in _Correct_ no longer matches up. Instead of removing this, could you replace the two first lines with something that actually does error?

Reply:
Thank you sincerely for your thoughtful review comments.

I have considered the example.I would be grateful to receive your valuable insights on this matter.